docs/release_guide.md

```diff
@@ -39,3 +65,12 @@ and any other administrative tasks completed on the repository.
 ## Announce the new release
 
 Make an announcement on the [mailing list](https://groups.google.com/g/openservicemesh).
+
+## Merge release branch into main
+
+Open a pull request from the release branch in the main repo (not fork) against the `main` branch.
+This should be done directly on Github by going to https://github.com/openservicemesh/osm/compare
+and opening a pull request from the release branch `release-<version>`.
+
+Always use the `Merge pull request` option once this pull request is ready for merge since this
+pull request must be a merge instead of a squash.
```

Comment:
Out of curiosity, why is this the case?

Reply:
Because multiple commits might exist and you don't want those to be squashed into one.

Reply:
The commit should be a merge commit actually, let me fix this part.